Formula/infer.rb

```diff
@@ -25,14 +24,12 @@ class Infer < Formula
   depends_on "automake" => :build
   depends_on "cmake" => :build
   depends_on "libtool" => :build
-  depends_on "ocaml" => :build
-  depends_on "ocaml-findlib" => :build
-  depends_on "ocaml-num" => :build
+  depends_on "ninja" => :build
   depends_on "opam" => :build
   depends_on "openjdk@8" => [:build, :test]
```

Comment:
Does it not work with a newer version?

Reply:
Good question -- @jvillard might know.

I am happy to try a local build with a more recent JDK and see what fails -- do you have a suggestion for a version that is more appropriate?

Reply:
In homebrew we always try for the latest version of software.

Reply:
Do you mean just dropping the `@8`?

Reply:
Infer supports Java11 too, we could try bumping the version of the openjdk dependency.

Reply:
Bumped it to `@11` -- testing locally.

Reply:
JDK11 worked locally; changed is pushed.